index.ejs

```diff
@@ -1,108 +1,93 @@
+    <% } %>
+  </head>
+  <body>
+    <% if (htmlWebpackPlugin.options.unsupportedBrowser) { %>
+    <style>.unsupported-browser { display: none; }</style>
+    <div class="unsupported-browser">
+      Sorry, your browser is not supported. Please upgrade to the latest version or switch your browser to use this
+      site. See <a href="http://outdatedbrowser.com/">outdatedbrowser.com</a> for options.
+    </div>
+    <% } %>
 
-<% for (var chunk in htmlWebpackPlugin.files.chunks) { %>
-<script src="<%= htmlWebpackPlugin.files.chunks[chunk].entry %>"></script>
-<% } %>
+    <% if (htmlWebpackPlugin.options.appMountId) { htmlWebpackPlugin.options.appMountIds.unshift(htmlWebpackPlugin.options.appMountId) } %>
+    <% for (item of htmlWebpackPlugin.options.appMountIds) { %>
+    <div id="<%= item %>"></div>
+    <% } %>
 
-<% if (htmlWebpackPlugin.options.devServer) { %>
-<script src="<%= htmlWebpackPlugin.options.devServer%>/webpack-dev-server.js"></script>
-<% } %>
+    <% if (htmlWebpackPlugin.options.window) { %>
+    <script type="text/javascript">
+      <% for (key in htmlWebpackPlugin.options.window) { %>
+      window['<%= key %>'] = <%= JSON.stringify(htmlWebpackPlugin.options.window[key]) %>;
+      <% } %>
+    </script>
+    <% } %>
 
-<% if (htmlWebpackPlugin.options.googleAnalytics) { %>
-<script>
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+    <% for (item of htmlWebpackPlugin.options.scripts) { %>
+    <script src="<%= item %>" type="text/javascript"></script>
+    <% } %>
 
+    <% for (key in htmlWebpackPlugin.files.chunks) { %>
+    <script src="<%= htmlWebpackPlugin.files.chunks[key].entry %>" type="text/javascript"></script>
+    <% } %>
 
-  <% if (htmlWebpackPlugin.options.googleAnalytics.trackingId) { %>
-    ga('create', '<%= htmlWebpackPlugin.options.googleAnalytics.trackingId%>', 'auto');
-    <% } else { throw new Error("html-webpack-template requires googleAnalytics.trackingId config"); }%>
+    <% if (htmlWebpackPlugin.options.devServer) { %>
+    <script src="<%= htmlWebpackPlugin.options.devServer %>/webpack-dev-server.js" type="text/javascript"></script>
+    <% } %>
 
-  <% if (htmlWebpackPlugin.options.googleAnalytics.pageViewOnLoad) { %>
-    ga('send', 'pageview');
-  <% } %>
-</script>
-<% } %>
-</body>
+    <% if (htmlWebpackPlugin.options.googleAnalytics) { %>
+    <script type="text/javascript">
+      !function(o,n,f,a,t,e,r){o.GoogleAnalyticsObject=t,o[t]=o[t]||function(){(o[t].q=o[t].q||[]).push(arguments)},o[t].l=1*new Date,e=n.createElement(f),r=n.getElementsByTagName(f)[0],e.async=1,e.src=a,r.parentNode.insertBefore(e,r)}(window,document,"script","//www.google-analytics.com/analytics.js","ga")
```

Comment:
Looks like it was already minimized and this just removed some linebreaks.  I think I'd rather keep the google-pasted code as it was to ensure we don't mess w/ its magic, esp since the benefit to the extra uglify is very little.